en/docs/develop-components/develop-components-with-git.md

```diff
@@ -0,0 +1,20 @@
+# Develop Components With Git
+
+Choreo enables you to develop components by connecting your GitHub repository. You have the flexibility to either connect an existing repository and use your current source code to create the component, or start with an empty repository and develop the source code later. By integrating your Git repositories with Choreo, you can automate tasks and optimize workflows across multiple systems, all within the Choreo platform. As of now, Choreo only supports GitHub as its Git provider. 
+
+In Choreo, you can connect a Git repository that contains Ballerina source code or a Docker project. To connect your Git repository to Choreo as a Docker project, your Git repository should contain the Dockerfile and the build's context. The Dockerfile specifies the instructions for building the Docker image, while the build context is a set of files located in the specified path used to build the image.
+
+Once you have connected your Git repository to Choreo, you can build, deploy, and manage your application easily. 
+
+## Authorize GitHub with Choreo 
+
+Authorizing Choreo as a GitHub application to GitHub provides Choreo the following permissions to perform the respective actions on your behalf within the repository:
```

Comment:
It would be better to say the user has to install the Choreo GitHub app.

```suggestion
When you install a GitHub Choreo App, it grants the app certain permissions to execute designated actions on your behalf within the repository.
```

Reply:
The issue with having the word install here is that we are not explicitly installing something.  We are actually just granting permission for Choreo to act on behalf of us.

Reply:
It is installed behind the scene in your organization. This happens in the authorization step for the very first time.